cli/lib/search.go

```diff
@@ -193,3 +201,38 @@ func versionsFromSearchedLibrary(library *rpc.SearchedLibrary) []*semver.Version
 	sort.Sort(semver.List(res))
 	return res
 }
+
+// indexNeedsUpdating returns whether library_index.json need updating.
+// A positive duration string must be provided to calculate the time threshold
+// used to update the index.
+// Valid duration units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
+// Use a duration of 0 to always update the index.
+func indexNeedsUpdating(duration string) bool {
```

Comment:
I'd like to change the `indexUpdateInterval` constant to a `time.Duration` like:
```go
const indexUpdateInterval = 60 * time.Minute
```
so the `indexNeedsUpdating` function may take directly a `time.Duration`

```suggestion
func indexNeedsUpdating(timeout time.Duration) bool {
```